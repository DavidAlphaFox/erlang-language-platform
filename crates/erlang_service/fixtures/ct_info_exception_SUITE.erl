-module(ct_info_exception_SUITE).

-export([all/0, groups/0]).
-export([a/1, b/1, c/1]).

all() -> throw({not_valid}).

groups() -> [{group_a, [b, c]}].

a(_Config) ->
    ok.

b(_Config) ->
    ok.

c(_Config) ->
    ok.
