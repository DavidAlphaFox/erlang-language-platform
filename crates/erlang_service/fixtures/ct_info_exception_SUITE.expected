Failed to fetch CT Info for "ct_info_exception_SUITE"